test/jdk/java/awt/Graphics/LCDTextAndGraphicsState.java

```diff
@@ -21,56 +21,84 @@
  * questions.
  */
 
-/**
+/*
  * @test
  * @bug 6576507
  * @summary Both lines of text should be readable
- * @run main/manual=yesno LCDTextAndGraphicsState
+ * @run main/manual LCDTextAndGraphicsState
  */
-import java.awt.*;
-import java.awt.geom.*;
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.GridBagLayout;
+import java.awt.GridBagConstraints;
+import java.awt.RenderingHints;
+import java.awt.AlphaComposite;
+import java.awt.GradientPaint;
+import java.awt.TextArea;
+import java.awt.Shape;
+import java.awt.Panel;
+import java.awt.Insets;
+
+import java.awt.event.ActionEvent;
+import java.awt.geom.RoundRectangle2D;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 public class LCDTextAndGraphicsState extends Component {
 
-    String text = "This test passes only if this text appears SIX TIMES";
+    private static final Frame instructionFrame = new Frame();
+    private static volatile boolean testResult = false;
+    private static volatile CountDownLatch countDownLatch;
+    private static String text = "This test passes only if this text appears SIX TIMES";
+    private static final String INSTRUCTIONS = "INSTRUCTIONS:\n\n" +
+            "You should see the text \' " + text + " \' on the frame. \n" +
+            "If you see the same then test pass else test fail.";
 
     public void paint(Graphics g) {
-
-        Graphics2D g2d = (Graphics2D)g.create();
+        Graphics2D g2d = (Graphics2D) g.create();
         g2d.setColor(Color.white);
-        g2d.fillRect(0,0,getSize().width, getSize().height);
+        g2d.fillRect(0, 0, getSize().width, getSize().height);
 
-        test1(g.create(0,   0, 500, 200));
+        test1(g.create(0, 0, 500, 200));
         test2(g.create(0, 200, 500, 200));
         test3(g.create(0, 400, 500, 200));
     }
 
     public void test1(Graphics g) {
-        Graphics2D g2d = (Graphics2D)g;
+        Graphics2D g2d = (Graphics2D) g;
         g2d.setRenderingHint(RenderingHints.KEY_TEXT_ANTIALIASING,
-                             RenderingHints.VALUE_TEXT_ANTIALIAS_LCD_HRGB);
+                RenderingHints.VALUE_TEXT_ANTIALIAS_LCD_HRGB);
```

Comment:
I'd rather keep them aligned as they were. I'm fine with reformatting, though. Both variants are acceptable and there's no strong preference to either in JDK code as far as I'm aware.

Reply:
fixed